client/base/Router.js

```diff
@@ -1,21 +1,21 @@
-import React, { PropTypes } from 'react'
-import { Router as ReactRouter } from 'react-router'
-import { FixNamedRoutesSupport as allowNames } from 'react-router-named-routes'
+import React from 'react'
+import { Router as ReactRouter, useRouterHistory } from 'react-router'
+import { syncHistoryWithStore } from 'react-router-redux'
+import useNamedRoutes from 'use-named-routes'
+import createHistory from 'history/lib/createBrowserHistory'
```

Comment:
Do we have to use the `history` library directly?  From my reading, there doesn't seem to be a reason that we can't use the singleton `browserHistory` from react-router like we were before.